css/style.css

```diff
@@ -1,19 +1,100 @@
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+/* header (nav) */
 
-body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+.nav_bar {
+  background-color: white;
+  position: fixed;
+  width: 100%;
+  height: 4.5rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
```

Comment:
Probably space between would have been more appropriate, and then using padding to move the items away from the edge.